test-result-summary-client/src/Dashboard/Widgets/Graph/ODM.jsx

```diff
@@ -9,14 +9,14 @@ import math from 'mathjs';
 import utils from './utils';
 
 const map = {
-    "Daily-ODM-all": "Daily-ODM Daily-ODM-Linux-PPCLE64 Daily-ODM-openJ9 Daily-ODM-zLinux Daily-ODM-zOS"
+    "Daily-ODM-all": "PerfNext-ODM-Child PerfNext-ODM-Master Daily-ODM Daily-ODM-Linux-PPCLE64 Daily-ODM-openJ9 Daily-ODM-zLinux Daily-ODM-zOS"

```

Comment:
Let's use Liberty instead of ODM since we are running Liberty twice a day so we should have enough data for testing. We stopped running ODM on Jenkins so you won't have real data and might be more work to just generate it. Unlike ODM, Liberty is integrated with TestKitGen and that's the style we're headed towards. Hence, it would be good to use Liberty runs. All widgets are fairly similar so it shouldn't be much work testing these changes in `DayTrader3.jsx`.